app/styles/components/link.css

```
@@ -14,14 +14,14 @@
 }
 
 .link--surface {
-    --link-color-override: currentColor;
+    --link-color-override: currentcolor;
      text-decoration: none;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)